examples/with-mongodb/README.md

```diff
@@ -0,0 +1,64 @@
+# MongoDB and Next.js Example
+
+This example shows how you can use a MongoDB database to support your Next.js application.
+
+## Demo
+
+### [https://with-mongodb-navy.now.sh/](https://with-mongodb-navy.now.sh/)
+
+**Pets** is an application that allows users to add their pets' information (e.g., name, owner's name, diet, age, dislikes, likes, and photo). They can also delete it or edit it anytime.
+
+## How to use
+
+### Using `create-next-app`
+
+Execute [`create-next-app`](https://github.com/zeit/next.js/tree/canary/packages/with-mongodb-app) with [npm](https://docs.npmjs.com/cli/init) or [Yarn](https://yarnpkg.com/lang/en/docs/cli/create/) to bootstrap the example:
+
+```bash
+npm init next-app --example with-mongodb with-mongodb-app
+# or
+yarn create next-app --example with-mongodb with-mongodb-app
+```
+
+## Configuration
+
+### Step 1. Connect MongoDB to the application
+
+Please see the [steps](./link-step.md) on how to connect MongoDB to your application
+
+### Step 2. Set up schema models for the application
+
+Based on the types of data needed for your application, you will modify the type definitions in [Pet.js](./models/Pet.js) as well as the seed data in [Pet-sampleSeed.json](./seed/Pet-sampleSeed.json)
+
+### Step 3. Import sample seed data to your MongoDB
+
+Please see the [steps](./data_import.md) on importing sample seed data into your MongoDB
+
+### Step 4. Set up environment variables
+
+Set `ROOT_URL` in [next.config.js](./next.config.js) to `http://localhost:3000` for quick testing locally, or set up your own development environment variables and use `process.env.ROOT_URL`. See the next.js [environement variable](https://nextjs.org/docs/api-reference/next.config.js/environment-variables) docs for more info.
```

Comment:
This can be handled by using dotenv support: https://nextjs.org/blog/next-9-4#new-environment-variables-support